config/resources/domain.json

```diff
@@ -265,58 +328,99 @@
         "temporal": {
           "type": "boolean",
           "predicate": "ext:temporal"
+        },
+        "variable-signage": {
+          "type": "boolean",
+          "predicate": "mobiliteit:variabeleSignalisatie"
         }
       },
-      "relationships":{
-        "zonality":{
+      "relationships": {
+        "zonality": {
           "predicate": "ext:zonality",
           "target": "skosConcept",
           "cardinality": "one"
+        },
+        "template": {
+          "predicate": "ext:template",
+          "target": "template",
+          "cardinality": "one"
+        },
+        "relatedTrafficSignConcepts": {
+          "predicate": "mobiliteit:heeftMaatregelconcept",
+          "target": "trafficSignConcept",
+          "cardinality": "many",
+          "inverse": true
         }
       },
       "new-resource-base": "http://data.lblod.info/traffic-measure-concepts/"
     },
-    "road-sign-concepts": {
-      "name": "roadSignConcept",
-      "class": "mobiliteit:Verkeersbordconcept",
-      "super": ["concept"],
+    "traffic-sign-concepts": {
+      "name": "trafficSignConcept",
+      "class": "mobiliteit:Verkeerstekenconcept",
+      "super": ["skosConcept"],
       "attributes": {
-        "image": {
-          "type": "string",
-          "predicate": "mobiliteit:grafischeWeergave"
-        },
         "meaning": {
           "type": "string",
           "predicate": "skos:scopeNote"
+        }
+      },
+      "relationships": {
+        "image": {
+          "predicate": "mobiliteit:grafischeWeergave",
+          "target": "image",
+          "cardinality": "one"
         },
-        "road-sign-concept-code": {
-          "type": "string",
-          "predicate": "skos:prefLabel"
+        "status": {
+          "predicate": "vs:term_status",
+          "target": "skosConcept",
+          "cardinality": "one"
+        },
+        "hasInstructions": {
+          "predicate": "mobiliteit:heeftInstructie",
+          "target": "template",
+          "cardinality": "many"
+        },
+        "relatedTrafficMeasureConcepts": {
```

Comment:
For some reason i had issues using snake case in relationships. Don't know why though, because i see in other repo's it works. Renamed it to `hasTrafficMeasureConcepts`